english/coffee/drink-tea/drink-hot-tea.md

```diff
@@ -5,3 +5,5 @@ You see a bowl of chocolates and sneakily eat one.
 
 [Everything around you seems to grow](alice/alice-in-wonderland.md)
 
+[Everything around you seems to shrink](../shrunk/shrunken.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.